test/functional/fixtures/api/es-next/hover/test.js

```diff
@@ -0,0 +1,26 @@
+var expect = require('chai').expect;
+
+describe('[API] Hover action', function () {
+    it('Should run hover over containers', function () {
+        return runTests('./testcafe-fixtures/hover-test.js', 'Hover over containers', { shouldFail: true, only: 'chrome' })
+            .catch(function (errs) {
+                expect(errs[0]).to.contains(' >  8 |    await t.hover(\'#container2\');');
```

Comment:
@AndreyBelym Should we check here the error message (`Hover on inputs raised`)? The test is ok now if any other error is raised? (for example `Hover selector is not applied to #container2`)